tensorflow/python/kernel_tests/conv_ops_test.py

```diff
@@ -327,11 +327,28 @@ def testConv2DKernelSmallerThanStrideValid(self):
                        expected=expected_output)
 
   def testConv2DKernelSmallerThanStrideSame(self):
-    expected_output = [1, 3, 7, 9]
+    #[[1 4 7 0]
```

Comment:
It's weird that the padding calculation computes a padding of 1 here -- without your changes to ops_util.cc, would all of these new tests still pass?